runtime/compiler/optimizer/VectorAPIExpansion.hpp

```diff
@@ -552,6 +639,12 @@ class TR_VectorAPIExpansion : public TR::Optimization
    */
    void invalidateSymRef(TR::SymbolReference *symRef);
 
+  /** \brief
+   *     Invalidared node so that it will not be vectorized or scalarized
```

Comment:
`Invalidared` in this comment looks like a typo.

Reply:
done